Makefile

```diff
@@ -251,3 +267,17 @@ test/scan: test/scan.c test/scan_tests.c $(ARLIB)
 
 test/split: test/split.c test/split_tests.c $(ARLIB)
 	$(CC) $(TEST_CFLAGS) -o $@ test/split.c $(ARLIB) $(TEST_LDFLAGS)
+
+prefix = /usr/local
+libdir = $(prefix)/lib
+includedir = $(prefix)/include
+
+install: $(ARLIB) $(SOLIB)
+	$(INSTALL) -d "$(DESTDIR)"$(libdir)
+	$(INSTALL) -m 644 $(ARLIB) "$(DESTDIR)"$(libdir)/$(ARLIB)
+	$(INSTALL) -m 755 $(SOLIB) "$(DESTDIR)"$(libdir)/$(SOLIBVER)
```

Comment:
Does the shared library actually need to be executable? (I would have thought 644 is sufficient here)

Reply:
Yes, shared libraries are always executable, just look into your `/usr/lib` (or `/usr/lib64`)…